src/Annotation/VoteUpDownWidget.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\vud\Annotation;
+
+use Drupal\Component\Annotation\Plugin;
+
+/**
+ * Defines a Vote Up/Down Widget annotation object.
+ *
+ * @see \Drupal\vud\Plugin\VoteUpDownFieldManager
+ * @see plugin_api
+ *
+ * @Annotation
+ */
+class VoteUpDownWidget extends Plugin {
+
+  /**
+   * The plugin ID.
+   *
+   * @var string
+   */
+  public $plugin_id;
+
+  /**
+   * The label of the plugin.
```

Comment:
I am thinking the following:
* summary line for the `$admin_label`: `Human readable name of the widget.`
* one example widget plugin definition:
  ```
  @VoteUpDownWidget(
  id = "alternate",
  admin_label = @Translation("Plain"),
  description = @Translation("Provides two arrows, up and down.")
  ```
Notice that both votes and widget template properties may not be needed on most widgets if we set the defaults, more about that on the next comments.